### PR TITLE
Update the INSTALL.md file to include steps for deploying al-folio on Netlify.

### PR DESCRIPTION
**Pull request by HenryF23**
Update CUSTOMIZE.md to include steps to deploy al-folio on Netlify.

### CODE REVIEW

**Review by george-gca — CHANGES_REQUESTED** [review]
Sorry for the delay on this. Can you move this to `INSTALL.md`? More specifically, above `Deployment to another hosting server (non GitHub Pages)`. I think it makes more sense there. Also, 2 more changes:
1. change branch `master` to `main`, since we are not using master anymore
2. make it as an enumeration. I think it is easier to follow

**Comment by HenryF23** [comment]
> Sorry for the delay on this. Can you move this to `INSTALL.md`? More specifically, above `Deployment to another hosting server (non GitHub Pages)`. I think it makes more sense there. Also, 2 more changes:
> 
> 1. change branch `master` to `main`, since we are not using master anymore
> 2. make it as an enumeration. I think it is easier to follow

I have moved the documentation to `INSTALL.md` and used enumeration for better readability. The branch should already be `main`

File: INSTALL.md [hunk 1]
```diff
@@ -129,6 +129,25 @@ If you keep your site on another branch, open `.github/workflows/deploy.yml` **o
 
 If you need to manually re-deploy your website to GitHub pages, go to Actions, click "Deploy" in the left sidebar, then "Run workflow."
 
+### Deploy on [Netlify](https://www.netlify.com/)
+
+1. [Use this template -> Create a new repository](https://github.com/new?template_name=al-folio&template_owner=alshedivat).
+2. Netlify: **Add new site** -> **Import an existing project** -> **GitHub** and give Netlify access to the repository you just created.
+3. Netlify: In the deploy settings
+    - Set **Branch to deploy** to `master`
```
Comment (george-gca) [inline INSTALL.md:137]: The branch here should be `main`.

**Review by george-gca — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 1 above)
Comment (HenryF23) [reply INSTALL.md]: > The branch here should be `main`.

I see. Thank you for pointing that out. It's now fixed.

**Review by george-gca — APPROVED** [review]

**Comment by george-gca** [comment]
Thanks for your contribution.

**Pull request merged**